ocw/tests/test_dap.py

```diff
@@ -17,11 +17,14 @@
 
 import unittest
 import datetime as dt
-import ocw.data_source.dap as dap
+try:
+    import ocw.data_source.dap as dap
+except ImportError:
```

Comment:
looks good @lewismc 🎆
This will probably solve the problem

Reply:
Are there any tests that need such skipping, let me know about it. I can work on it then